frontend/src/services/objectService.ts

```diff
@@ -29,6 +29,17 @@ export default {
     return comsAxios().delete(`${PATH}/${objectId}`);
   },
 
+  /**
+   * @function listObjects
+   * List and search for all objects
+   * @returns {Promise} An axios response
+   */
+  getMetadata(headers: any = {}, params: any = {}) {
+    // remove objId array if its first element is undefined
+    if (params.objId && params.objId[0] === undefined) delete params.objId;
+    return comsAxios().get(`${PATH}/metadata`, { headers: headers, params: params });
```

Comment:
i think this path has changed now to GET /object/metadata?objectId=abc,def,hij
so i guess from your store you can just call it once for the set of objects